webapp/src/main/resources/config/application.properties

```
@@ -130,7 +130,9 @@ l10n.PluralFormUpdater=true
 # Spring session
 #l10n.spring.session.store-type=jdbc
 
-# Opengrok server configuration, to generate links, etc
-#l10n.opengrok.server=https://opengrok.someplace.com
-#l10n.opengrok.repositoryMapping.mojitoName=opengrokName
-#l10n.opengrok.repositoryMapping.More\ complicated\ name=opengrokName2
\ No newline at end of file
+# Link configuration, to generate links, etc
+#l10n.link.locationMapping.mojitoName=https://opengrok.someplace.com
+#l10n.link.commitMapping.More\ complicated\ name=https://github.com
+#l10n.link.extractedPrefix=/some/extractor/prefix/
```

Hm, relooking at it, since `extracted prefix` and `tagDelimiter` is repository specific, I'd have it that way

I'd have something like: 
```suggestion
l10n.link.{repository}.location.url=https://opengrok.someplace.com/xref/{openGrokRepository}
l10n.link.{repository}.location.extractedPrefix=/some/extractor/prefix/
l10n.link.{repository}.commit.url=http://githbu.com/{gitHubRepository}xyz

l10n.link.{repository2}.location.url=https://someotherplace.com/xref/{textUnitName}
```